dmarc.tf

```diff
@@ -0,0 +1,9 @@
+resource "aws_route53_record" "dmarc" {
+  provider = aws.route53
+
+  name    = "_dmarc.${var.domain}"
+  records = ["v=DMARC1;p=reject;sp=reject"]
```

Comment:
Not being familiar with DMARC, so we want this to be the default for a variable or will it pretty much always be this?